action.yml

```
@@ -8,6 +8,10 @@ inputs:
   circleci-token:
     description: 'A personal API token to CircleCI'
     required: true
+  should-wait:
```

Perhaps
```suggestion
  wait-for-completion:
```
or something along those lines.